### ISSUE TITLE
[Bug] Process Memory Provider doesn't work. Linux.

### ISSUE THREAD

**Issue by codecnotsupported** [WerWolv/ImHex#1515]
### Operating System

Linux

### What's the issue you encountered?

The Process Memory Provider doesn't display process contents.

### How can the issue be reproduced?

Open any process with the Process Memory Provider.

### ImHex Version

1.32.2

### ImHex Build Type

- [ ] Nightly or built from sources

### Installation type

Appimage

### Additional context?

This includes GDB as well.
I don't think this is a permission error as I used `echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope` to ensure proper permission.
Note: 
https://github.com/WerWolv/ImHex/blob/390c1469b1b463e0cfd2f5cf68344ed27b255f6f/plugins/builtin/source/content/providers/process_memory_provider.cpp#L66
Note: 
https://github.com/WerWolv/ImHex/blob/390c1469b1b463e0cfd2f5cf68344ed27b255f6f/plugins/builtin/source/content/providers/process_memory_provider.cpp#L66
It may fail due to insufficient permissions.  
A modal informing the user, or a button that fixes it would be nice.
I have no idea how to elevate privileges to set the proper permissions in a DE neutral way. 
`pkexec setcap cap_sys_ptrace=eip `program_path`` followed by a program restart might work, but I'm unsure how this interacts with appimages as they contain the executable.

**Comment by SkyLeite** [comment]
I don't think this is a permissions issue, as running the Windows version through Wine leads to the same issue. Worth noting that this is the same environment in which Cheat Engine works as expected.

**Comment by paxcut** [comment]
This should really be a feature request because it just hasn't been implemented yet. There are several mentions of this in discord server. It is bound to get implemented at some point, but if somebody is looking for features to implement they may miss this one because is mislabeled as a bug.

**Comment by wrb2012** [comment]
I find some library to scan memory https://github.com/brkzlr/libscanmem-PINCE
